README.md

```diff
@@ -76,6 +76,13 @@ will be added as well. Modules which depend on other libraries
 
 There are many ways to support Cats' development:
 
+ * Fix bugs: Despite using static types, law-checking, and
+   property-based testing bugs can happen. Reporting problems you
+   encounter (with the documentation, code, or anything else) helps us
+   to improve. Look for issues labelled "ready" as good targets. but 
+   *please add a comment to the issue* and label the issue "In progress"
```

Comment:
I don't think non-committers can change labels. I think **bold** might be called for here rather than *italics* since it has become an issue already.